diagnostic.md

```diff
@@ -177,5 +209,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+The the first media queiry . stretch should size to  25px, the second to 100px and the third media query to 30% of 800. Responseive Design : )
```

Comment:
137.5 px - Reason being that since it's min-width, the media query will have ```.stretch``` be 25% and at 550px width, 25% is 137.5 px.